webapp/utils/text_formatting.jsx

```diff
@@ -162,12 +162,23 @@ export function autolinkAtMentions(text, tokens, usernameMap) {
         return (Constants.SPECIAL_MENTIONS.indexOf(u) !== -1 || Boolean(usernameMap[u]));
     }
 
+    function isSpecialMention(username) {
+        return Constants.SPECIAL_MENTIONS.reduce((acc, val) => {
+            return acc || username === val;
+        }, false);
+    }
+
     function addToken(username, mention) {
         const index = tokens.size;
         const alias = `$MM_ATMENTION${index}`;
 
+        let tokenValue = `<span data-mention='${username}'><a class='mention-link' href='#'>${mention}</a></span>`;
+        if (isSpecialMention(username)) {
```

Comment:
And then you can also remove this part so `addToken` won't even be called for `@all` and the other special mentions